erp/export/schemas.py

```diff
@@ -0,0 +1,61 @@
+from dataclasses import dataclass
+from typing import Literal, Set
+
+
+@dataclass
+class OfficialSchema:
```

Comment:
I'm also realizing we are duplicating the schema definition while we have it defined as JSON in https://github.com/MTES-MCT/acceslibre-schema… Wouldn't it make sens to all this while parsing the `schema.json`? Redefining it programmatically here may introduce a risk of desynchronization between the two definitions.